smartdashboard/utils/ManifestReader.py

```diff
@@ -183,10 +186,27 @@ def create_filereader(path: str) -> ManifestFileReader:
         manifest_file_reader = ManifestFileReader(path)
     except FileNotFoundError as fnf:
         raise ManifestError(
-            title="Manifest file does not exist.", file=path, exception=fnf
+            title="Manifest file does not exist.", file=str(path), exception=fnf
         ) from fnf
     except json.decoder.JSONDecodeError as jde:
         raise ManifestError(
-            title="Manifest file could not be decoded.", file=path, exception=jde
+            title="Manifest file could not be decoded.", file=str(path), exception=jde
         ) from jde
     return manifest_file_reader
+
+
+def get_manifest_path(directory: t.Optional[pathlib.Path]) -> pathlib.Path:
+    """Get the manifest path using the directory
+    path passed in from the command line arguments.
+
+    :param directory: An experiment directory
+    :type directory: t.Optional[pathlib.Path]
+    :return: Manifest path
+    :rtype: pathlib.Path
+    """
+
+    if directory is not None:
+        manifest_path = directory / ".smartsim/telemetry/manifest.json"
+    else:
+        manifest_path = pathlib.Path()
```

Comment:
Hmmmmm really good questions. I was intentionally pointing it to the CWD so that it would raise the `Manifest Error`, but adding `pathlib.Path() / ".smartsim/telemetry/manifest.json"` would do that anyway. I think I'd rather the error be raised during the creation of the `ManifestReader`, rather than returning the path though.